src/app/components/notifier/mute-notifier.service.js

```diff
@@ -27,6 +30,16 @@
       info("Trying to say something? You are muted.");
     }
 
+    function joinedMuted(){
+      info("You're muted. Unmute yourself to join the conversation!");
```

Comment:
Yes I thought about "You're muted because there are more than X participants in this room. Unmute yourself..." but I wanted to use a brief sentence to avoid wrapping of the notification on small screens. I will try to make a better sentence.